File: array/outsideSeason.java
package springIntro;

import java.util.*;

public class outsideSeason {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		String month [] = {"Ocak", "Subat", "Mart", "Nisan", "Mayis", "Haziran", "Temmuz", 
				"Agustos", "Eylul", "Ekim", "Kasim", "Aralik"};
		System.out.println("Enter a month number");
		int choose = in.nextInt();
		if(choose > 12)
			System.out.println("Error");
		else
			System.out.println(month[choose - 1]);
	}

}

File: array/outsideSeason.cpp
#include<iostream>
using namespace std;
int main(){
	char month[][20] = {"Ocak", "Subat", "Mart", "Nisan", "Mayis", "Haziran", "Temmuz", 
	"Agustos", "Eylul", "Ekim", "Kasim", "Aralik"};
	int choose = 0;
	cout << "Enter a month number = ";
	cin >> choose;
	if(choose > 12){
		cout << "Error";
	}
	else{
		cout << month[choose - 1];
	}
	return 0;
}
